local-server.js

```diff
@@ -0,0 +1,13 @@
+var http = require('http');
```

Comment:
This file is a little http server in  node.js that roughly simulates being served from github pages. That is, if the path is exactly '/', serve the index.html, otherwise, serve 404.html. Of course, github pages does a LOT more, but this works for quick and easy testing, and doesn't require anything other than node.js installed: https://nodejs.org/en/download/current/